conjur/cli.py

```diff
@@ -186,8 +190,28 @@ def run(self, *args):
         # *************** LIST COMMAND ***************
 
         # pylint: disable=line-too-long
-        resource_subparsers.add_parser('list',
-                                       help='List all available resources belonging to this account')
+        list_subparser = resource_subparsers.add_parser('list',
```

Comment:
IMHO literals should come from a resource file

Reply:
WDTM?

Reply:
It means all string are i a resource files and you are referring to an entry in the file.